packages/webcert/src/feature/certificate/Buttons/RevokeCertificateModalContent.tsx

```diff
@@ -89,7 +83,7 @@ export const RevokeCertificateModalContent: React.FC<Props> = ({ onChange, type
           <div>
             <p className="iu-fw-bold iu-fs-200">
               <MandatoryIcon additionalStyles={mandatoryIconAdditonalStyles} display={textArea.value.length < 1} />
-              Ange orsaken till felet.
+              Ange orsaken till felet.{' '}
```

Comment:
Remove if possible. (According to previous conversation, this is added by prettier)